update NAV_CONTROLLER_OUTPUT comment

dagar opened the pull request:
PX4 NAV_CONTROLLER_OUTPUT implemented here - https://github.com/PX4/Firmware/pull/4363

I'd personally rather have altitude/altitude_sp and airspeed/airspeed_sp instead of altitude/airspeed error. Any objection to adding the actual setpoints to this message as well?

LorenzMeier commented:
Can we just strip project names entirely? I'm trying to curate MAVLink to be platform neutral and don't want anything PX4 (or APM) specific in the common message set.

dagar commented:
Done.

dagar commented:
@LorenzMeier I updated the comment as requested.

I also added the actual altitude/airspeed setpoints for tuning. It's a bit redundant next to the errors, but that's what people are used to and the aspd_error and alt_error fields are used by ardupilot.

LorenzMeier commented:
We can't add fields without breaking the message. So I have to reject the PR in the current shape.

dagar commented:
Oh okay, I was hoping it would be okay to slap them on the end. Additional fields removed.
